### ISSUE TITLE
Internal server error for: "/v1/actions/views/overview" on server with rbac enabled 

### ISSUE THREAD

**Issue by humblearner** [StackStorm/st2#3941]
```bash
$ curl -X GET -H  'Connection: keep-alive' -H  'Accept-Encoding: gzip, deflate' -H  'Accept: */*' -H  'User-Agent: python-requests/2.14.2' -H  'X-Auth-Token: <>' http://127.0.0.1:9101/v1/actions/views/overview
{
    "faultstring": "Internal Server Error"
}
```

```
2018-01-12 13:38:54,084 140549577661968 INFO logging [-] 14ec3912-274b-4935-808f-2274197cf4d0 - GET /v1/actions/views/overview with query={} (remote_addr='127.0.0.1',method='GET',request_id='14ec3912-274b-4935-808f-2274197cf4d0',query={},path='/v1/actions/views/overview')
2018-01-12 13:38:54,091 140549577661968 AUDIT auth [-] Token with id "5a5925084066ae2192caf352" is validated.
2018-01-12 13:38:54,105 140549577661968 ERROR router [-] Failed to call controller function "get_all" for operation "st2api.controllers.v1.actionviews:overview_controller.get_all": 'NoneType' object has no attribute 'name'
Traceback (most recent call last):
  File "/opt/stackstorm/st2/local/lib/python2.7/site-packages/st2common/router.py", line 450, in __call__
    resp = func(**kw)
  File "/opt/stackstorm/st2/local/lib/python2.7/site-packages/st2api/controllers/v1/actionviews.py", line 136, in get_all
    raw_filters=raw_filters)
  File "/opt/stackstorm/st2/local/lib/python2.7/site-packages/st2api/controllers/resource.py", line 381, in _get_all
    requester_user=requester_user)
  File "/opt/stackstorm/st2/local/lib/python2.7/site-packages/st2api/controllers/resource.py", line 163, in _get_all
    limit = validate_limit_query_param(limit=limit, requester_user=requester_user)
  File "/opt/stackstorm/st2/local/lib/python2.7/site-packages/st2api/controllers/resource.py", line 436, in validate_limit_query_param
    user_is_admin = rbac_utils.user_is_admin(user_db=requester_user)
  File "/opt/stackstorm/st2/local/lib/python2.7/site-packages/st2common/rbac/utils.py", line 249, in user_is_admin
    is_system_admin = user_is_system_admin(user_db=user_db)
  File "/opt/stackstorm/st2/local/lib/python2.7/site-packages/st2common/rbac/utils.py", line 269, in user_is_system_admin
    return user_has_role(user_db=user_db, role=SystemRole.SYSTEM_ADMIN)
  File "/opt/stackstorm/st2/local/lib/python2.7/site-packages/st2common/rbac/utils.py", line 287, in user_has_role
    user_role_dbs = rbac_services.get_roles_for_user(user_db=user_db)
  File "/opt/stackstorm/st2/local/lib/python2.7/site-packages/st2common/services/rbac.py", line 97, in get_roles_for_user
    queryset = UserRoleAssignment.query(user=user_db.name)
AttributeError: 'NoneType' object has no attribute 'name'
2018-01-12 13:38:54,111 140549577661968 ERROR error_handling [-] API call failed: 'NoneType' object has no attribute 'name'
Traceback (most recent call last):
  File "/opt/stackstorm/st2/local/lib/python2.7/site-packages/st2common/middleware/error_handling.py", line 46, in __call__
    return self.app(environ, start_response)
  File "/opt/stackstorm/st2/local/lib/python2.7/site-packages/st2common/middleware/streaming.py", line 47, in __call__
    return self.app(environ, start_response)
  File "/opt/stackstorm/st2/local/lib/python2.7/site-packages/st2common/router.py", line 499, in as_wsgi
    resp = self(req)
  File "/opt/stackstorm/st2/local/lib/python2.7/site-packages/st2common/router.py", line 454, in __call__
    raise e
AttributeError: 'NoneType' object has no attribute 'name' (_exception_data={},_exception_class='AttributeError',_exception_message="'NoneType' object has no attribute 'name'")
2018-01-12 13:38:54,116 140549577661968 INFO logging [-] 14ec3912-274b-4935-808f-2274197cf4d0 - 500 46 31.51ms (content_length=46,request_id='14ec3912-274b-4935-808f-2274197cf4d0',runtime=31.51,remote_addr='127.0.0.1',status=500,method='GET',path='/v1/actions/views/overview')
```

Seems like related to: https://github.com/StackStorm/st2/pull/3939

**Comment by Kami** [comment]
Thanks for reporting this, will look into it.

**Comment by Kami** [comment]
Resolved by #3943.
